tracer/src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -434,6 +447,12 @@ internal static bool IsAlpine()
                 envVars["LD_PRELOAD"] = ldPreload;
             }
 
+            if (!string.IsNullOrEmpty(tracerProfilerArm64))
+            {
+                envVars["CORECLR_PROFILER_PATH_ARM64"] = tracerProfilerArm64;
+                envVars["COR_PROFILER_PATH_ARM64"] = tracerProfilerArm64;
```

Comment:
I'm still confused with [how this plays with x64/Arm64EC vs Arm64](https://learn.microsoft.com/en-us/windows/arm/arm64ec)... AFAICT from that we can't mix ARM64EC and native arm64 binaries, so can we actually profile a native arm64 binary if we're arm64ec compiled? 🤔

Reply:
We can merge both ARM64EC + ARM64, that's what they call [`ARM64X`](https://learn.microsoft.com/en-us/windows/arm/arm64x-pe), that's what we are building here.


>When a user builds a project, Visual Studio would normally compile for Arm64EC and then link the outputs into an Arm64EC binary. When BuildAsX is set to true, Visual Studio will instead compile for both Arm64EC and Arm64. The Arm64EC link step is then used to link both together into a single Arm64X binary. The output directory for this Arm64X binary will be whatever the output directory is set to under the [Arm64EC configuration](https://learn.microsoft.com/en-us/windows/arm/arm64ec-build).